pkg/webui/components/form/index.js

```diff
@@ -60,13 +60,13 @@ const Form = props => {
     validateOnChange,
     validateOnMount,
     validateSync,
-    validationContext,
+    validationContext: initialValidationContext,
     validationSchema,
-    stripUnusedFields,
```

Comment:
Instead of using the prop, we can now just use the third argument of the `onSubmit()`-callback.

```js
const handleSubmit useCallback((values, { resetForm }, cleanedValues) => { … }, […])
```